### PR TITLE
Fixing Webview not able to Render Second Brain

### PR DESCRIPTION
**Pull request by MathRoda**
https://user-images.githubusercontent.com/92212925/205989941-d5b26b62-df76-4907-bd7a-1d8d0329c677.mp4

first attempt to solve this issue #77 by implementing [JCEF Maven - apache-2.0 License](https://github.com/jcefmaven/jcefmaven) 
in  the video above, i used static button for the sake of  simulating the flow for opening second brain in Sphinx Mac app

note : second brain take a bit of time load even in my own browser idk if it has something todo with where the host server located

@tomastiminskas

### CODE REVIEW

File: common/src/desktopMain/kotlin/chat/sphinx/webview/CefBrowser.kt [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                CefApp.getInstance().dispose()
+                dispose()
+            }
+        })
+    }
+
+    companion object {
+        const val SECOND_BRAIN = "https://second-brain.sphinx.chat"
+        const val GOOGLE = "https://www.google.com"
+        const val MY_GITHUB = "https://github.com/MathRoda"
```
Comment (kevkevinpal) [inline common/src/desktopMain/kotlin/chat/sphinx/webview/CefBrowser.kt:80]: we should be able to get this data from tribe, it should be app_url @tomastiminskas not sure if you can help

Comment (MathRoda) [reply common/src/desktopMain/kotlin/chat/sphinx/webview/CefBrowser.kt:80]: yeah ofc this just for testing will be cleared after review

Comment (kevkevinpal) [reply common/src/desktopMain/kotlin/chat/sphinx/webview/CefBrowser.kt:80]: perfect, lemme know when you want me to review again 🚀

**Comment by MathRoda** [comment]
@kevkevinpal @tomastiminskas please when you have time test the changes on Mac or Linux and let me know if they work fine.

**Comment by tomastiminskas** [comment]
@MathRoda I will test tomorrow for sure

**Comment by tomastiminskas** [comment]
@MathRoda @kevkevinpal  I was testing. Definitely the hardest part is done because the Second Brain page is loading inside Sphinx on KMM app, which is great. Here are some issues I found:

- When tapping the icon to open the browser, it's taking more than 1/2 minutes until it opens. In that time, the console is showing this message:
`Dec 07, 2022 10:14:48 AM me.friwi.jcefmaven.impl.progress.ConsoleProgressHandler handleProgress` with a download progress. 
This happens just the first time so it seems some process is being performed that first time. Maybe this can be avoided

- When closing the CefBrowser window the app is entirely closed.

The delay on Second Brain load seems to be on that page, because when testing with google.com it load pretty fast. 
If we can find a workaround for the 2 issues above, we can move on to showing the icon just for tribes with appUrl and using it on the browser, and finally implementing the bridge to the browser which is essential for this feature

**Comment by kevkevinpal** [comment]
> @MathRoda @kevkevinpal I was testing. Definitely the hardest part is done because the Second Brain page is loading inside Sphinx on KMM app, which is great. Here are some issues I found:
> 
>     * When tapping the icon to open the browser, it's taking more than 1/2 minutes until it opens. In that time, the console is showing this message:
>       `Dec 07, 2022 10:14:48 AM me.friwi.jcefmaven.impl.progress.ConsoleProgressHandler handleProgress` with a download progress.
>       This happens just the first time so it seems some process is being performed that first time. Maybe this can be avoided
> 
>     * When closing the CefBrowser window the app is entirely closed.
> 
> 
> The delay on Second Brain load seems to be on that page, because when testing with google.com it load pretty fast. If we can find a workaround for the 2 issues above, we can move on to showing the icon just for tribes with appUrl and using it on the browser, and finally implementing the bridge to the browser which is essential for this feature

I think the delay in starting second brain is ok for now as it might be an issue on the frontend since it takes sometime to load on the mac app aswell but this is looking good

**Comment by tomastiminskas** [comment]
@MathRoda I can try compiling the app with the bundle included and see how bigger the app files become, then we can decide. Let me know if you can push the bundle to a separated branch and I can compile from there to test

**Comment by MathRoda** [comment]
@tomastiminskas yeah for sure, also i could use your help on finding a way to control the browser visibility i'm blocked

**Comment by tomastiminskas** [comment]
@MathRoda for sure. I will take a look at it tomorrow to see if I find a workaround

**Comment by kevkevinpal** [comment]
just linking this issue
https://github.com/stakwork/sphinx-kotlin-ui/issues/69

**Pull request merged**